loopy/isl_helpers.py

```diff
@@ -83,6 +83,13 @@ def make_slab(space, iname, start, stop, step=1):
 
```

Comment:
I don't think the inequality
```
``start <= step*iname < stop``.
```
works for negative step.

Reply:
No longer allowed in  9c1b2ba.